src/utils/status/host/hostStatus.js

```diff
@@ -18,3 +23,5 @@ export const getHostStatus = host => {
 export const getSimpleHostStatus = host => getHostStatus(host).status;
 
 export const canHostAddMachine = host => [HOST_STATUS_READY].includes(getSimpleHostStatus(host));
+export const canHostStartMaintenance = hostNode => hostNode && !getNodeUnschedulable(hostNode);
+export const canHostStopMaintenance = hostNode => hostNode && getNodeUnschedulable(hostNode);
```

Comment:
no need for `hostNode && ` in `canHostStopMaintenance` as getNodeUnschedulable returns false as default

Reply:
Done